templates/aws/fips.html

```diff
@@ -329,6 +331,10 @@ <h2>FIPS-certified Ubuntu components for AWS</h2>
             </tr>
           </tbody>
         </table>
+        <hr class="p-rule--muted" />
+        <p>
+          * The AWS kernel variant is based on the generic Ubuntu 22.04 kernel, which is certified under <a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4894">NIST Certificate #4894</a>
+        </p>
```

Comment:
Not sure we want to use a `p` here - apparently, it sets a max-width of 40em that makes it look weird on large screens, with an unnecessary line wrap:

![image](https://github.com/user-attachments/assets/6f7c9d46-7d52-4c7a-9483-faf11228670a)

Maybe we should use a `div` instead?

![image](https://github.com/user-attachments/assets/aa43a6f0-4775-4e32-8748-ad82e3625743)